source/NeutronaWand/Configuration.h

```diff
@@ -79,7 +79,7 @@ const uint8_t VOLUME_EFFECTS_MULTIPLIER = 5;
  * Part #: BL28Z-3005SA04Y
  * Only compatible with the gpstar Neutrona Wand board, and not a Arduino Nano.
 */
-bool b_28segment_bargraph = false;
+bool b_28segment_bargraph = true;
```

Comment:
This just came up as a change--was this intentional as the new norm, or just part of other testing?

Reply:
ah damn, that should be false. I will force change it.